agent/consul/stream/proto_types.go

```diff
@@ -0,0 +1,76 @@
+package stream
+
+import (
+	"encoding/json"
+
+	proto "github.com/golang/protobuf/proto"
+)
+
+type UntypedMap map[string]interface{}
```

Comment:
What is this? It's not generated? I guess we maybe use this to represent opaque maps somehow but why is it not protobuf generated and why is it JSON encoding? Maybe some doc comments would clarify!